mysite/account/tests.py

```diff
@@ -287,26 +303,26 @@ def test_edit_email_address(self):
         assert new_email not in paulproteus_page
 
 
+# list of photos
 photos = [os.path.join(os.path.dirname(__file__),
           '..', '..', 'sample-photo.' + ext)
           for ext in ('png', 'jpg')]
 
 
 def photo(f):
+    """ Helper function for photo file """
     filename = os.path.join(os.path.dirname(__file__), '..', f)
     assert os.path.exists(filename)
     return filename
 
 
-@skipIf(not mysite.base.depends.Image, ("Skipping photo-related tests "
-                                        "because PIL is missing. Look in "
-                                        "ADVANCED_INSTALLATION.mkd for "
-                                        "information."))
```

Comment:
@ehashman A reasonable concern but now that pillow is in requirements.txt and pip is much more stable than before with its use of wheels. Let's take it out for now. If it becomes an issue later, we can always put it back in. If you feel strongly that we should leave in, I'm happy to do that too. Just let me know.